webapp/src/app/chat/messages/messages.component.html

```diff
@@ -0,0 +1,42 @@
+<div #chatMessagesContainer class="messages space-y-4 overflow-y-auto" style="max-height: 100%">
+  <div *ngFor="let message of messageHistory" class="flex w-full" [ngClass]="{ 'justify-end': message.sender === 'USER', 'justify-start': message.sender === 'SYSTEM' }">
+    <div class="flex space-x-2 md:w-3/5" [ngClass]="{ 'flex-row-reverse': message.sender === 'USER' }">
+      <div *ngIf="message.sender === 'USER'" class="ml-2 flex flex-col">
+        <hlm-avatar>
+          <img [src]="'https://github.com/' + user()!.username + '.png'" [alt]="user()?.name + '\'s avatar'" hlmAvatarImage />
+          <span hlmAvatarFallback>
+            {{ user()?.name?.slice(0, 2)?.toUpperCase() ?? '?' }}
+          </span>
+        </hlm-avatar>
+      </div>
+
+      <div *ngIf="message.sender === 'SYSTEM'" class="mr-2 flex flex-col">
+        <div class="w-10 h-10 bg-transparent border-2 border-cyan-500 rounded-full flex items-center justify-center">
+          <lucide-angular [img]="Hammer" class="size-6 text-cyan-500"></lucide-angular>
+        </div>
+      </div>
+
+      <div
+        class="flex flex-col space-y-2"
+        [ngClass]="{
+          'text-right': message.sender === 'USER',
```

Comment:
In ChatGpt and Whatsapp all text is always alligned to the left. Left alligned text looks weird I think